### PR TITLE
Add missing signature to interact()

### PR DESCRIPTION
**Pull request by MidnightDesign**
See http://interactjs.io/docs/#selector-contexts

### CODE REVIEW

File: interactjs/interact-tests.ts [hunk 1]
```diff
@@ -11,7 +11,9 @@ var rectangle: ClientRect = {
     bottom: 100,
     height: 100
 };
-var interactable = interact(button);
+let context = document.createElement('a');
+let interactable = interact('.foo', {context: context});
```
Comment (adidahiya) [inline interactjs/interact-tests.ts]: please use double quotes for consistency within the file.

**Comment by adidahiya** [comment]
looks good aside from the minor linting nit

**Comment by thasner** [comment]
👍

**Comment by MidnightDesign** [comment]
Good catch. Thanks! Quotes are fixed.

**Pull request merged**